src/device-state/db-format.ts

```diff
@@ -37,9 +37,9 @@ export async function getApp(id: number): Promise<InstancedApp> {
 export async function getApps(): Promise<InstancedAppState> {
 	const dbApps = await getDBEntry();
 	const apps: InstancedAppState = {};
-	for (const app of dbApps) {
-		apps[app.appId] = await buildApp(app);
-	}
+	await Promise.all(
+		dbApps.map(async (app) => (apps[app.appId] = await buildApp(app))),
```

Comment:
I think this would be clearer as
```suggestion
		dbApps.map(async (app) => {
			apps[app.appId] = await buildApp(app);
		}),
```